Task.js

```diff
@@ -102,6 +108,51 @@ class Task extends Unit {
     );
   }
 
+  isURL(source) {
+    try {
+      new URL(source);
+      return true;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  jsPath() {
+    let final_paths = [];
+    for (let js_path of this.js_modules) {
+      if (this.isURL(js_path)) {
+        final_paths.push(js_path);
+        continue;
+      }
+
+      const absolute_path = path.resolve(
+        path.join(Config.build_path(this.exp_path), this.basedir, js_path)
+      );
+      final_paths.push(
+        path.relative(path.dirname(this.targetPath()), absolute_path)
+      );
+    }
+    return final_paths;
+  }
+
+  cssPath() {

```

Comment:
Both jsPath and cssPath can be the same function with final_path and js_path/css_path passed in as parameters.

Reply:
A new function is created by the name finalPath that takes 1 parameter